LBH.AdultSocialCare.Api/V1/Controllers/HomeCareApproveBrokered/HomeCareApproveBrokeredController.cs

```diff
@@ -0,0 +1,36 @@
+using LBH.AdultSocialCare.Api.V1.Boundary.HomeCareApproveBrokeredBoundary.Response;
+using LBH.AdultSocialCare.Api.V1.UseCase.HomeCareApproveBrokeredUseCase.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LBH.AdultSocialCare.Api.V1.Controllers.HomeCareApproveBrokered
+{
+    [Route("api/v1/home-care-approve-brokered")]
+    [Produces("application/json")]
+    [ApiController]
+    [ApiExplorerSettings(GroupName = "v1")]
+    [ApiVersion("1.0")]
+    public class HomeCareApproveBrokeredController : BaseController
+    {
+        private readonly IGetHomeCareApproveBrokeredUseCase _getHomeCareApproveBrokeredUseCase;
+
+        public HomeCareApproveBrokeredController(IGetHomeCareApproveBrokeredUseCase getHomeCareApproveBrokeredUseCase)
+        {
+            _getHomeCareApproveBrokeredUseCase = getHomeCareApproveBrokeredUseCase;
+        }
+
+        /// <summary>Gets the specified home care approve package brokered identifier.</summary>
+        /// <param name="homeCarePackageId">The home care package identifier.</param>
+        /// <returns>The home care approve brokered response.</returns>
+        [HttpGet]
+        [Route("{homeCarePackageId}")]
```

Comment:
(Being a child route in home care hierarchy, `{homeCarePackageId}` can be moved to the top)